modules/osd-create-cluster-ccs.adoc

```diff
@@ -151,9 +151,23 @@ To successfully create a cluster, you must select *Enable Secure Boot support fo
 +
 endif::osd-on-gcp[]
 .. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+.. Optional: Expand *Advanced Encryption* to make changes to encryption settings.
+ifdef::osd-on-aws[]
+.. Optional: Accept the default setting *Use default KMS Keys* to use your default AWS KMS key, or select *Use Custom KMS keys* to use a custom KMS key.
+... Optional: Enter the AWS Key Management Service (KMS) custom key Amazon Resource Name (ARN) ARN in the *Key ARN* field.
+// ifdef::osd-on-gcp[]
```

Comment:
Not sure if you meant to comment this out.

Reply:
> Not sure if you meant to comment this out.

No- great catch, thanks!